be/src/olap/task/engine_storage_migration_task.cpp

```diff
@@ -409,8 +409,9 @@ Status EngineStorageMigrationTask::_copy_index_and_data_files(
                 InvertedIndexStorageFormatPB::V1) {
                 for (const auto& index : tablet_schema.inverted_indexes()) {
                     auto index_id = index->index_id();
-                    auto index_file =
-                            _tablet->get_segment_index_filepath(rowset_id, segment_index, index_id);
+                    auto index_file = InvertedIndexDescriptor::get_index_file_path_v1(
+                            InvertedIndexDescriptor::get_index_file_path_prefix(segment_file_path),
+                            index_id, "");
```

Comment:
use index suffix instead of "", although we disable variant index for v1

Reply:
done

Reply:
no index suffix here